src/app/screens/SignUp/index.tsx

```diff
@@ -2,98 +2,91 @@ import React from 'react';
             required: { value: true, message: i18next.t('SignUp:required') }
           })}
+          error={errors?.lastName?.message}
         />
-        {errors.last_name && <label className={styles['error-labels']}>{errors.last_name.message}</label>}
-        <label className={styles['custom-labels']}>{i18next.t('SignUp:email')}</label>
-        <input
-          name="email"
-          className={styles['custom-inputs']}
-          type="email"
-          ref={register({
+        <label className={styles.customLabels}>{i18next.t('SignUp:email')}</label>
+        <CustomInput
+          name={SIGN_UP_FIELDS.email}
+          inputRef={register({
             required: { value: true, message: i18next.t('SignUp:required') },
             pattern: {
               value: /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i,
               message: i18next.t('SignUp:invalidEmail')
             }
           })}
+          error={errors?.email?.message}
+          type="email"
         />
-        {errors.email && <label className={styles['error-labels']}>{errors.email.message}</label>}
-        <label className={styles['custom-labels']}>{i18next.t('SignUp:password')}</label>
-        <input
-          name="password"
-          type="password"
-          className={styles['custom-inputs']}
-          ref={register({
+        <label className={styles.customLabels}>{i18next.t('SignUp:password')}</label>
+        <CustomInput
+          name={SIGN_UP_FIELDS.password}
+          inputRef={register({
             required: { value: true, message: i18next.t('SignUp:required') }
           })}
-        />
-        {errors.password && <label className={styles['error-labels']}>{errors.password.message}</label>}
-        <label className={styles['custom-labels']}>{i18next.t('SignUp:confirmPassword')}</label>
-        <input
-          name="password_confirmation"
-          className={styles['custom-inputs']}
+          error={errors?.password?.message}
           type="password"
-          ref={register({
+        />
+        <label className={styles.customLabels}>{i18next.t('SignUp:confirmPassword')}</label>
+        <CustomInput
+          name={SIGN_UP_FIELDS.passwordConfirmation}
+          inputRef={register({
             required: { value: true, message: i18next.t('SignUp:required') },
             validate: value => value === watch('password') || (i18next.t('SignUp:passwordsNoMatch') as string)
           })}
+          error={errors?.passwordConfirmation?.message}
+          type="password"
         />
-        {errors.password_confirmation && (
-          <label className={styles['error-labels']}>{errors.password_confirmation.message}</label>
-        )}
-        <button className={`${styles['custom-button']} ${styles['signup-button']}`} type="submit">
+        <button className={`${styles.customButton} ${styles.signupButton}`} type="submit">
           {i18next.t('SignUp:signUp')}
         </button>
         <hr className={styles.separator} />
```

Comment:
don't use `<hr`, if you need a separator use css pseudoelemnt like `before` or `after` in the closest component